sql/2020/09_Performance/median_lcp_score_by_month.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+# Median of Largest contentful paint 75% score by month
+
+SELECT
+  date,
+  device,
+  median_p75_lcp
+FROM (
+  SELECT
+    date,
+    device,
+    PERCENTILE_CONT(p75_lcp, 0.5) OVER(PARTITION BY date, device) AS median_p75_lcp
+  FROM
+    `chrome-ux-report.materialized.device_summary`
+  WHERE
+    date >= "2019-08-01"
```

Comment:
```suggestion
    date >= "2019-08-01"
    AND date <= "2020-08-01"
```

Reply:
sorry @rviscomi what are you referring to specifically when asking for resolution? the dates?